apps/common/behaviours/generate-pdf.js

```diff
@@ -0,0 +1,53 @@
+'use strict';
+
+const pdf = require('html-pdf');
+const UploadModel = require('../../common/models/file-upload');
+const concat = require('concat-stream');
+const path = require('path');
+
+module.exports = superclass => class extends superclass {
+  saveValues(req, res, callback) {
+    super.saveValues(req, res, (err, values) => {
```

Comment:
Specifically in this case because one of the other behaviours with a `saveValues` will be to submit the application to iCasework, and that definitely needs to happen *after* this has completed.